core/src/web-runtime.ts

```diff
@@ -13,7 +13,7 @@ export class CapacitorWeb {
     // Gracefully degrade in non-Proxy supporting engines, e.g. IE11. This
     // effectively means that trying to access an unavailable plugin will
     // locally throw, but this is still better than throwing a syntax error.
-    if ('Proxy' in window) {
+    if ('Proxy' in (typeof window === 'undefined' ? global : window)) {
```

Comment:
```suggestion
    if (typeof Proxy !== 'undefined') {
```

Reply:
Just tested it in the sample project (https://codesandbox.io/s/github/flogy/gatsby-starter-capacitor). Building works fine and it also still runs in IE 11 browser (which was the initial problem fixed by https://github.com/ionic-team/capacitor/pull/2759).